includes/class-wc-stripe-payment-method-configurations.php

```diff
@@ -114,10 +114,17 @@ private static function set_payment_method_configuration_cache( $configuration )
 	 * @return object|null
 	 */
 	private static function get_payment_method_configuration_from_stripe() {
+		// Only allow fetching payment configuration once per minute.
+		$fetch_cooldown = get_option( 'wcstripe_payment_method_config_fetch_cooldown', 0 );
+		if ( $fetch_cooldown > time() ) {
+			return null;
+		}
+
 		$result         = WC_Stripe_API::get_instance()->get_payment_method_configurations();
 		$configurations = $result->data ?? null;
 
 		if ( ! $configurations ) {
+			update_option( 'wcstripe_payment_method_config_fetch_cooldown', time() + MINUTE_IN_SECONDS );
```

Comment:
We set the cooldown when there are no configurations (Stripe limit is already hit). Instead shouldn't we always set the cooldown to prevent multiple calls within a minute?

Reply:
I think Eric and Anne had the same concern over Slack. Can you check the latest changes they suggested, please? I think that may fix the issue